src/components/AppList/AppList.tsx

```diff
@@ -20,17 +21,24 @@ class AppList extends React.Component<IAppListProps> {
     return (
       <section className="AppList">
         <header className="AppList__header">
-          <h1>Applications</h1>
+          <div className="row padding-t-big">
```

Comment:
Add the `align-center` class to center the button vertically and a `collapse-*` class to make it responsive